.github/workflows/check-roles.yml

```diff
@@ -0,0 +1,34 @@
+name: "check-roles"
+
+on:
+  schedule:
+    - cron: "0 0 * * *" # every day at midnight
+
+jobs:
+  check-roles:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 18
+          cache: "yarn"
+
+      - name: Install Foundry
+        uses: foundry-rs/foundry-toolchain@v1
+        with:
+          version: nightly
+
+      - name: generate roles files
+        run: forge script CheckRoles
+        env:
+          CHAIN_IDS: "1,42161,137,100,43114,8453,56,42220,1101,10,59144"
```

Comment:
It would be better to fetch those chains via th sdk or something to not miss any/ not have to update the workflow

Reply:
not really possible considering we need to add every RPC urls